charts/pokt-portal-v2/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
We just need to make sure we don't put secrets in connection strings. Ideally this config.json only holds hostnames and we put secrets in env vars @adshmh @crisog